CreeDictionary/CreeDictionary/templatetags/creedictionary_extras.py

```diff
@@ -29,19 +63,32 @@ def orth(sro_original: str):
 
         <span lang="cr" data-orth
               data-orth-latn="wâpamêw"
-              data-orth-latn="wāpamēw"
+              data-orth-latn-x-macron="wāpamēw"
               data-orth-cans="ᐚᐸᒣᐤ">wâpamêw</span>
     """
 
-    sro_circumflex = conditional_escape(sro_original)
-    sro_macrons = conditional_escape(to_macrons(sro_original))
-    syllabics = conditional_escape(sro2syllabics(sro_original))
+    sro_circumflex = sro_original
+    sro_macrons = to_macrons(sro_original)
+    syllabics = sro2syllabics(sro_original)
+
+    if orthography == "Latn":
+        inner_text = sro_circumflex
+    elif orthography == "Latn-x-macron":
+        inner_text = sro_macrons
+    elif orthography == "Cans":
+        inner_text = syllabics
+    else:
+        raise ValueError(f"invalid orthography: {orthography}")
 
-    return mark_safe(
+    return format_html(
```

Comment:
Django's documentation recommends this over using `conditional_escape()` and `mark_safe()`: https://docs.djangoproject.com/en/3.0/ref/utils/#django.utils.html.format_html